index.html

```diff
@@ -44,5 +45,62 @@ <h3>LET’S CONNECT</h3>
       </ul>
     </div>
   </section>
+  <!--Works section-->
+
+  <section class="work-section">
+    <div class="card">
+      <img src="assets/images/tonic1.svg" alt="tonic1">
+      <div class="tonic">
+        <h1>Tonic</h1>
+        <ul>
+          <li><a class="Canopy" href="#">Canopy <i class="fa fa-circle"></i></a></li>
+          <li><a href="#"> Back End Dev <i class="fa fa-circle"></i></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Kindly, I have noticed that you have only added `one card` while the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=fKSSzvYlyU1Biln3-0) has `4 cards`, I think it would be better to include all the project cards to replicate the design as much as possible😎😎

   **Note:** Using a pre-designed template can save you time and effort in the design process, as you don't have to start from scratch. This can also help you get your project up and running more quickly 👍 👍